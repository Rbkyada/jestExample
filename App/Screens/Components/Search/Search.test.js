import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

// Note: test renderer must be required after react-native.
import Search from './Search';

jest.mock('react-native-localize', () => {});
jest.mock('react-native-linear-gradient', () => {});
jest.mock('react-native-config', () => {});

it('renders Snapshot correctly', () => {
  const snap = renderer.create(<Search />).toJSON();
  expect(snap).toMatchSnapshot();
});
