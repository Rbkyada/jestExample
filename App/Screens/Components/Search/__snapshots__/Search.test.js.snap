// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Snapshot correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      },
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      },
      Object {
        "backgroundColor": undefined,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 18,
        },
        Object {
          "color": undefined,
        },
      ]
    }
    xlarge={true}
  >
    Search Tab
  </Text>
</RCTSafeAreaView>
`;
